mobile/src/screens/chat/components/chat-item/chat-item.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
```

Comment:
let's create a common component from this called Card, because we will use it across the app

Reply:
Done